packages/web-shared/components/ContentSeriesSingle.js

```diff
@@ -205,7 +205,6 @@ function ContentSeriesSingle(props = {}) {
                 gridTemplateColumns={{
                   _: 'repeat(1, minmax(0, 1fr));',
                   md: 'repeat(2, minmax(0, 1fr));',
```

Comment:
We don't need to make this change on the ContentSeriesSingle.js, just the ContentSingle.js component. 

The request was because a content single is narrower for long form content, but also displays sibling cards if the content item is part of a series.

Reply:
Whoops, shouldn't have jumped the gun on that one  😅 should be fixed now!